### PR TITLE
Update German.json

### PR DESCRIPTION
**Pull request by iilonmasc**
Thank you for proof-reading and updating some fields with better values for the specific context @manveru0

### CODE REVIEW

File: init/MUOS/language/German.json [hunk 1]
```diff
@@ -285,20 +285,20 @@
...
     "Instant Shutdown": "Sofortiges Herunterfahren",
-    "Sleep 10s + Shutdown": "Ruhezustand 10s + Shutdown",
-    "Sleep 30s + Shutdown": "Ruhezustand 30s + Shutdown",
-    "Sleep 60s + Shutdown": "Ruhezustand 60s + Shutdown",
-    "Sleep 2m + Shutdown": "Ruhezustand 2m + Shutdown",
-    "Sleep 5m + Shutdown": "Ruhezustand 5m + Shutdown",
-    "Sleep 10m + Shutdown": "Ruhezustand 10m + Shutdown",
-    "Sleep 30m + Shutdown": "Ruhezustand 30m + Shutdown",
-    "Sleep 60m + Shutdown": "Ruhezustand 60m + Shutdown",
+    "Sleep 10s + Shutdown": "Ruhezustand 10s + Herunterfahren",
```
Comment (xonglebongle) [inline init/MUOS/language/German.json:288]: This might have to be shortened as this string will be quite long and will overlap the right hand side label.

Comment (iilonmasc) [reply init/MUOS/language/German.json:288]: We are currently discussing whether to use anglicisms instead for Shutdown / Standby / Sleep and so on to shorten the strings.

**Comment by manveru0** [comment]
@xonglebongle: just to be sure: "sleep function" (line 246) is gonna replace the present title "Sleep Shutdown" on the left side? So "Sleep suspend" and "Instant Shutdown" are gonna be corresponding values?

**Comment by xonglebongle** [comment]
> @xonglebongle: just to be sure: "sleep function" (line 246) is gonna replace the present title "Sleep Shutdown" on the left side? So "Sleep suspend" and "Instant Shutdown" are gonna be corresponding values?

That is correct :smile:

**Pull request merged**